### ISSUE TITLE
[Programmatic Access - Make code - Home Page]: Accessible name is not defined for the 'Link' present under ''Plant watering' button.

### ISSUE THREAD

**Issue by vroopea** [microsoft/pxt-microbit#6087]
### User Experience:
Visually impaired people who depend on screen reader will get impacted if accessible name is not defined for the interactive elements present on the page. Due to which, the end users will not be able to know the purpose and cannot access it efficiently.

**Note:**
User credentials should NOT be included in the bug.

### Repro Steps:

1. Launch application URL: https://makecode.microbit.org/beta#
2. Home page will open.
3. Navigate to 'Science' section and expand it.
4. Turn on the AI (Accessibility insights for web) and observe whether accessible name is defined for the 'Link' or not.

### Actual Result:
Issue: Accessible name is not defined for the 'Link' present under ''Plant watering' button.

### Expected Result:
Accessible name should be defined for the 'Link' present under ''Plant watering' button.

**Example:** Aria-label="Show instructions" can be defined for the link.

### Issue: 
Ensures links have discernible text (link-name - [https://accessibilityinsights.io/info-examples/web/link-name](https://accessibilityinsights.io/info-examples/web/link-name))

### Target application: 
Microsoft MakeCode for micro:bit - [https://makecode.microbit.org/beta#](https://makecode.microbit.org/beta#)

### Element path: 
.attached[href$="plant-watering"][target="_blank"]:nth-child(1)

### Snippet: 
`<a class="ui icon link button attached" href="/projects/plant-watering" target="_blank" rel="noopener noreferrer" role="presentation" tabindex="-1"><i class="icon file text " aria-hidden="true" role="presentation"></i></a>`

### How to fix: 
**Fix any of the following:**

- Element does not have text that is visible to screen readers
- aria-label attribute does not exist or is empty
- aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
- Element has no title attribute

### MAS Reference:
[MAS 4.1.2 – Name, Role, Value](https://aka.ms/MAS4.1.2)

### ARIA Authoring Ref Link/Fluent Patterns Guidance /HITS Guidance:
NA

### UIA/DOM details available:
Yes (Refer attachment)

### Reference Links:

- [Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.
- External Bug Process: If this bug belongs to an external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information, please use this [External Bug Process (sharepoint.com)](https://microsoft.sharepoint.com/teams/cea11y/SitePages/External-Bug-Process.aspx?xsdata=MDN8MDF8fDE1MThmM2JlZGFjMDRlZDhiZmViYTUxMDA2M2NiZWI3fDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDF8MHw2Mzc3NDEyNTM2MTEzNzIwNzR8R29vZHxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaUlpd2lRVTRpT2lJaUxDSlhWQ0k2TVRKOQ%3D%3D&sdata=MnlpSXlVMnpGT3cvZ0xJNnFWUWZWTGt6dmo1VkJMZFpIZHBvUUduNDhyMD0%3D&ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2cv-kgeetala%40microsoft.com). “HCL Staff should not log any third-party external bugs. Should be routed to EDAD team”.
- Please reach out to [C&AI Teams channel](https://apc01.safelinks.protection.outlook.com/?url=https://teams.microsoft.com/l/team/19%253a3cd071821b694f11b054d878453cd75f%2540thread.skype/conversations?groupId%3D30e3f968-15dd-4da2-bd8e-85f27e4af2c3%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=02%7c01%7cAklavyaM%40hcl.com%7c24e89c6ac26f42cfe40c08d7667e49f3%7c189de737c93a4f5a8b686f4ca9941912%7c0%7c0%7c637090564906844600&sdata=22JjBMoA1ID/KnYqFa/OB%2BrkTaLh5YYsGBd/7Bm7u5w%3D&reserved=0) for any process-related queries.

### Test Environment:

- **OS Version:** Windows 11 24H2 (OS Build:26100.3037)
- **Browser:** Chrome Version 132.0.6834.196 (Official Build) (64-bit)
- **Test Matrix:** Chrome + NVDA
- **URL:** [https://makecode.microbit.org/.](https://makecode.microbit.org/beta#)

### Attachment:

<img width="1353" alt="Image" src="https://github.com/user-attachments/assets/bcb09c8f-54db-4cde-b4a9-922d59311697" />

**Comment by vroopea** [comment]
#Accessibility; #A11yAuto; #K4W; #A11yMAS; #A11ySev2; #MAS4.1.2; #HCL; #Benchmark; #Win11-Chrome; #BM-HCL-MakeCode-Feb2025

**Comment by Yamini234u** [comment]
As checked issue on https://www.microsoft.com/en-us/makecode/teach. The issue has been fixed. Hence, closing the bug
